sisl/viz/plots/tests/test_bands.py

```diff
@@ -16,13 +16,16 @@
 import numpy as np
```

Comment:
Instead of importing xarray here, you should do something like is done in `sisl/io/siesta/tests/test_bands.py` with importorskip